tasks/cluster-setup-pcs-0.10.yml

```diff
@@ -0,0 +1,22 @@
+- name: Create a corosync.conf file content using pcs-0.10
+  command:
+    cmd: >
+      pcs cluster setup
+      --corosync_conf {{ __ha_cluster_tempfile_corosync_conf.path | quote }}
+      --overwrite
+      {{ ha_cluster_cluster_name | quote }}
+      {% for node in ansible_play_hosts %}
```

Comment:
I don't think this is a task that we can test in our CI - in CI, the ansible hostname is the path and filename of the platform.qcow2 e.g. `/cache/centos8.qcow2`

note that the vpn role has a similar problem - it needs to test setting up configuration of tunnels with actual hostnames.  This was solved by having the test change the inventory_hostname, and using add_host to create some dummy hosts.
see https://github.com/linux-system-roles/vpn/blob/master/tests/tasks/setup_test.yml and https://github.com/linux-system-roles/vpn/blob/master/tests/tasks/add_hosts.yml - they are called from the test like this: https://github.com/linux-system-roles/vpn/blob/master/tests/tests_host_to_host_cert.yml#L12
however - I'm not sure if that changes `ansible_play_hosts` - in your testing you might have to overwrite `ansible_play_hosts` with your "fake" hostnames.

Reply:
Hopefully, I managed to resolve this - fedora-32/ansible-2.9 and fedora-33/ansible-2.9 are passing now.